src/main/java/seedu/address/model/person/exceptions/DuplicateAssignmentException.java

```diff
@@ -4,8 +4,8 @@
  * Signals that the operation will result in duplicate Persons (Persons are considered duplicates if they have the same
  * identity).
  */
-public class DuplicatePersonException extends RuntimeException {
-    public DuplicatePersonException() {
+public class DuplicateAssignmentException extends RuntimeException {
+    public DuplicateAssignmentException() {
         super("Operation would result in duplicate persons");
     }
 }
```

Comment:
Similar to the other comment, the comments for the method and the string parameter at line 9 use 'persons' instead of 'assignments'.